spec.md

```diff
@@ -547,7 +555,7 @@ In addition, the authors leave currently open the following details:
 * Whether and how more detailed "forensic reporting" should be accomplished, as
   discussed in the section _Failure_ _Reporting_.
 
-* The registration of the .well-known/smtp-sts URI as per [@!RFC5785].
+* The registration of the .well-known policy._smtp-sts URI prefix as per [@!RFC5785].
```

Comment:
To be a nitpicker, we don't need to register the *prefix*: RFC 5785 concerns only .well-known suffix registration. So the hostname on the policy domain is up to us and requires no registration. ;)